modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/cards/Cards.js

```diff
@@ -80,6 +88,21 @@ const Card = ({item, schema}) => {
 							openSidePanel,
 						});
 					}
+					else {
```

Comment:
If I'm not wrong, we could remove the condition to check for `onActionDropdownItemClick` and pass it directly as a parameter of the new `handleActionClick`, as it already takes care of it (see [handleActionClick.ts#120](https://github.com/liferay-frontend/liferay-portal/pull/4109/files#diff-6e51e0408d91b38b7626f9a5e22acc64548afc1378ea6646b8f93437cc48fa34R120)).

One odd thing about the `onActionDropdownItemClick` callback is that it only provides support for some context methods, like `openSidePanel` and `loadData` but not others like `openModal`.

Reply:
yep, I was a bit hesitant to remove it because we don't have a stance being it an API or not. I think we can remove it though, and just keep the same handler for every view. If a click handler customization is needed then we will provide a client extension.

After a second thought, yes, I'll remove it